tests/tck/features/agg/Agg.feature

```diff
@@ -0,0 +1,207 @@
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dst             | age  |
+      | "Tony Parker"   | 36.0 |
+      | "Manu Ginobili" | 41.0 |
+
+  Scenario: Implicit GroupBy
+    When executing query:
+      """
+      GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst, $$.player.age AS age
+      | YIELD $-.dst AS dst, 1+avg(distinct $-.age) AS age, abs(5) as abs
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dst             | age  | abs |
+      | "Tony Parker"   | 37.0 | 5   |
+      | "Manu Ginobili" | 42.0 | 5   |
+    When executing query:
+      """
+      GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst, $$.player.age AS age
+      | YIELD $-.dst AS dst, 1+avg(distinct $-.age) AS age where $-.age > 40
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dst             | age  |
+      | "Manu Ginobili" | 42.0 |
+    When executing query:
+      """
+      $var=GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst, $$.player.age AS age;
+      YIELD $var.dst AS dst, (INT)abs(1+avg(distinct $var.age)) AS age
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dst             | age |
+      | "Tony Parker"   | 37  |
+      | "Manu Ginobili" | 42  |
+    When executing query:
+      """
+      $var1=GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst;
+      $var2=GO FROM "Tim Duncan" OVER serve YIELD serve._dst AS dst;
+      YIELD $var1.dst AS dst, count($var1.dst) AS count
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dst             | count |
+      | "Tony Parker"   | 1     |
+      | "Manu Ginobili" | 1     |
+
+  Scenario: Error Check
+    When executing query:
+      """
+      YIELD avg(*)+1 ,1+2 ,(INT)abs(min(2))
+      """
+    Then a SemanticError should be raised at runtime: Could not apply aggregation function `AVG(*)` on `*`
+    When executing query:
+      """
+      GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst, $$.player.age AS age
+      | GROUP BY $-.dst YIELD avg(distinct $-.age) AS age
+      """
+    Then a SemanticError should be raised at runtime:  GroupBy list must in Yield list
+    When executing query:
+      """
+      GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst, $$.player.age AS age
+      | GROUP BY $-.dst,$-.age YIELD $-.age,avg(distinct $-.age) AS age
+      """
+    Then a SemanticError should be raised at runtime: GroupBy item `$-.dst` must be in Yield list
+    When executing query:
+      """
+      GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst, $$.player.age AS age
+      | GROUP BY $-.dst,$-.x YIELD avg(distinct $-.age) AS age
+      """
+    Then a SemanticError should be raised at runtime:  `$-.x', not exist prop `x'
+    When executing query:
+      """
+      GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst, $$.player.age AS age
```

Comment:
Can you add test about this, expect execute ok
```
 GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst, $$.player.age AS age
      | GROUP BY $-.age+1 YIELD $-.age+1, avg(distinct $-.age) AS age
```

Reply:
Already added.